packages/fxa-content-server/app/scripts/views/sign_in_unblock.js

```diff
@@ -42,6 +43,71 @@ const View = FormView.extend({
     });
   },
 
+  showValidationError(el, err, shouldFocusEl = true) {
```

Comment:
Since this view extends the base Form [view](https://github.com/mozilla/fxa/blob/26acd623417b6082734cb65b4ec35cc6336b0069/packages/fxa-content-server/app/scripts/views/form.js#L383) it would have access to the `showValidationError` function. You can remove it here.